docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,66 @@
+  * i.e. in the Selection Filters section
+  * i.e. in the Metadata column of the table
+  * i.e. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * i.e. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * i.e. In the **Imaging Browser** module
+    * i.e. Click the **Upload** tab
+  * You can also bold when defining features in a list, to separate the term being defined from the definition itself, i.e. when describing columns in a table:<br>
+     **Column heading 1**: description<br>
+     **Column heading 2**: description<br>
+     Etc.
+     
+## Tone
+
+Yes, LORIS is used for high-level research and yes, the people using LORIS are very smart and yes, we want people in the scientific community to take LORIS seriously. At the same time, the easier any user finds a text to read, the more they’ll retain and use the information. And a friendly tone will help them understand that LORIS is built and maintained by humans. There’s a lot of research that shows people will engage and have more patience with a piece of software when they’re reminded that there’s more than just a computer behind it. Using a more conversational tone will subtly have this effect. We’re not talking about humour and exclamation marks—just small tactics that make content as easy to understand as possible.
+
+Please note, this is NOT about “dumbing things down”. It’s simply about using the most direct language and tone to get the information across to the user in the least amount of time, with the least amount of friction. The quicker a user understands something, the happier they are, the more trust they have with the software, and the less silly questions you can expect to get to the dev mailing list! 
+
+* Use second person point of view rather than third
+  * i.e. “This module allows *you* to browse…” (rather than “This module allows *users* to browse…”)
+* Keep it concise. Avoid unnecessary information.
+  * Try not to be repetitive. 
+  * i.e. the Help text for the Imaging Browser module begins with the title Imaging Browser. You don’t need to start text with “In the Imaging Browser module...”, because the user already knows the module based on the title. Instead, keep it short and say “In this module…”
+* Assume the user is tech-savvy enough in the year 2020 to know about features that are intuitive or common across modern platforms, so you don’t need to repeat in every Help text
+  * i.e. clicking the column headers to re-sort a table
+  * i.e. clicking a Download button to download something
+  * i.e. navigate pages of a table using the arrows provided
+* Use [Plain Language](https://www.plainlanguage.gov/resources/articles/elements-of-plain-language/)
+  * As a guideline, refer to [this list](https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/) of words to avoid, and alternate suggestions 
```

Comment:
I don't like that we refer to foreign government rules. Not only theses can change without us noticing but they may not be the best in our situation.

Reply:
@PapillonMcGill I think we could still include these links for context(inspiration), without saying that we are strictly following them -- would you be ok with that?